examples/avro-cli.py

```diff
@@ -113,16 +129,19 @@ def produce(topic, conf):
     producer.flush()
 
 
-def consume(topic, conf):
+def consume(topic, conf, avro_serializer):
     """
         Consume User records
     """
-    from confluent_kafka.avro import AvroConsumer
-    from confluent_kafka.avro.serializer import SerializerError
+    from confluent_kafka.serialization import SerializerError
+
+    print("Consuming user records from topic {} with group {}. ^c to exit.".format(
+        topic, conf["group.id"]))
 
-    print("Consuming user records from topic {} with group {}. ^c to exit.".format(topic, conf["group.id"]))
+    c = Consumer(conf,
+                 key_serializer=StringSerializer('utf_8'),
```

Comment:
what does a consumer need a serializer for?

Reply:
Each record is associated with a utf_8 encoded string

https://github.com/confluentinc/confluent-kafka-python/pull/787/files/b4f2481a7afeccbdcc156d43693d8171b1f12d3e#diff-cc5f63fc6235c090a21b390a38e66ba0R108

Reply:
Wouldn't that be a Deserializer?

Reply:
ack, also *Serializer has been renamed *Serde. There is some protobuf stuff mixed in the serializer tests so I'll push those changes soon